### PR TITLE
feat(FOROME-243): add confirmation modal to delete dtree

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-243

In addition:
1. Remove apply, cancel buttons for deleting tree operation (now there is only confirmation modal)

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/utils/filter-refiner/compareConditionsForJoin.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  //     difference(condition[index], startConditions[index]).length === 0 &&
+  //     condition[condition.length - 1].length ===
+  //       startConditions[startConditions.length - 1].length
+  //   ) {
+  //     return true
+  //   }
+  //   return false
+  // })
+
+  console.log(toJS(currentConditions), toJS(startConditions))
```
Comment (AlMaQntr) [inline src/utils/filter-refiner/compareConditionsForJoin.ts]: I think you pushed some intermediate version of code

File: src/utils/filter-refiner/compareConditionsForJoin.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    (acc, condition: any[], index): number => {
+      if (difference(condition[index], startConditions[index]).length === 0) {
+        acc += 1
+      }
+      return acc
+    },
+    0,
+  )
+
+  // const someResult = currentConditions.every((condition: any[], index) => {
```
Comment (AndreyUstyumenko) [inline src/utils/filter-refiner/compareConditionsForJoin.ts]: comments?

Comment (AndreyUstyumenko) [inline src/utils/filter-refiner/compareConditionsForJoin.ts]: comments?

Comment (Lapk1n) [reply src/utils/filter-refiner/compareConditionsForJoin.ts]: removed

Comment (Lapk1n) [reply src/utils/filter-refiner/compareConditionsForJoin.ts]: removed

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by AlMaQntr — APPROVED** [review]

**Pull request merged**